libs/domain/picking/src/components/customer-note-modal/customer-note-modal.component.ts

```diff
@@ -0,0 +1,32 @@
+import { html, LitElement, TemplateResult } from 'lit';
+import { property } from 'lit/decorators.js';
+import { styles } from './customer-note-modal.styles';
+
+export class CustomerNoteModal extends LitElement {
+  static styles = styles;
+
+  @property() customerNote?: string;
+
+  protected closeCustomerNoteModal(): void {
+    this.dispatchEvent(new Event('close'));
```

Comment:
Like and public event it should have name like: `oryx.[...]`.

And could you double check, please. `oryx.close` event that is emitted by oryx-modal should bubble up and you can listen to it outside, instead `close` event